src/components/Option.js

```diff
@@ -33,6 +33,8 @@ export type OptionProps = PropsWithStyles &
     /* Type is used by the menu to determine whether this is an option or a group.
     In the case of option this is always `option`. */
     type: 'option',
+    /* The data of the option rendered in the Option component */
+    data: any
```

Comment:
Do you know why that generic type has been removed? It'd be nice to have that type here 🙂 

![image](https://user-images.githubusercontent.com/38978723/47004196-4e2d6280-d131-11e8-8448-e24c1e6fb595.png)